switch with empty last clause

The following program `sample.go` triggers a panic:

```go
package main

func main() {
    a := 2
    switch {
    case a == 1:
        println(1)
    case a == 2:
        println(2)
    default:
    }
    println("bye")
}
```

Expected result:
```console
$ go run ./sample.go
2
bye
```

Got:
```console
$ yaegi ./sample.go
./sample.go:5:2: CFG post-order panic: runtime error: index out of range [-1]
goroutine 1 [running]:
runtime/debug.Stack(0x1, 0xc00038c000, 0x40)
        /home/marc/sdk/go1.14.2/src/runtime/debug/stack.go:24 +0x9d
github.com/containous/yaegi/interp.(*Interpreter).Eval.func1(0xc00021be00)
        /home/marc/src/yaegi/interp/interp.go:320 +0xbc
panic(0xd796a0, 0xc000280680)
        /home/marc/sdk/go1.14.2/src/runtime/panic.go:969 +0x166
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2.1(0xc000380b00)
        /home/marc/src/yaegi/interp/cfg.go:398 +0xb3
panic(0xd6f120, 0xc000284080)
        /home/marc/sdk/go1.14.2/src/runtime/panic.go:969 +0x166
github.com/containous/yaegi/interp.(*node).lastChild(...)
        /home/marc/src/yaegi/interp/cfg.go:1942
github.com/containous/yaegi/interp.(*Interpreter).cfg.func2(0xc000380b00)
        /home/marc/src/yaegi/interp/cfg.go:1513 +0xc5b3
github.com/containous/yaegi/interp.(*node).Walk(0xc000380b00, 0xc00021bca8, 0xc00021bc78)
        /home/marc/src/yaegi/interp/interp.go:191 +0xaf
github.com/containous/yaegi/interp.(*node).Walk(0xc000380700, 0xc00021bca8, 0xc00021bc78)
        /home/marc/src/yaegi/interp/interp.go:188 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000380200, 0xc00021bca8, 0xc00021bc78)
        /home/marc/src/yaegi/interp/interp.go:188 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000380000, 0xc00021bca8, 0xc00021bc78)
        /home/marc/src/yaegi/interp/interp.go:188 +0x70
github.com/containous/yaegi/interp.(*Interpreter).cfg(0xc000368000, 0xc000380000, 0x7ffcf6d0e629, 0x1b, 0xc000366048, 0x4, 0x7ffcf6d0e629, 0x1b, 0x0)
        /home/marc/src/yaegi/interp/cfg.go:55 +0x160
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc000368000, 0xc000376000, 0x9b, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/marc/src/yaegi/interp/interp.go:343 +0x1ea
main.main()
        /home/marc/src/yaegi/cmd/yaegi/yaegi.go:139 +0x554

```

--- a/_test/switch35.go
+++ b/_test/switch35.go
@@ -1,0 +1,17 @@
+package main
+
+func main() {
+	a := 2
+	switch {
+	case a == 1:
+		println(1)
+	case a == 2:
+		println(2)
+	default:
+	}
+	println("bye")
+}
+
+// Output:
+// 2
+// bye

--- a/_test/switch36.go
+++ b/_test/switch36.go
@@ -1,0 +1,14 @@
+package main
+
+func main() {
+	a := 2
+	switch {
+	case a == 1:
+		println(1)
+	case a == 2:
+	}
+	println("bye")
+}
+
+// Output:
+// bye

--- a/_test/switch37.go
+++ b/_test/switch37.go
@@ -1,0 +1,15 @@
+package main
+
+func main() {
+	a := 2
+	switch {
+	case a == 1:
+		println(1)
+	case a == 3:
+	default:
+	}
+	println("bye")
+}
+
+// Output:
+// bye

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -1510,24 +1510,29 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 			for i := l - 1; i >= 0; i-- {
 				c := clauses[i]
 				c.gen = nop
-				body := c.lastChild()
-				if len(c.child) > 1 {
-					cond := c.child[0]
-					cond.tnext = body.start
-					if i == l-1 {
-						setFNext(cond, n)
+				if len(c.child) == 0 {
+					c.tnext = n
+					c.fnext = n
+				} else {
+					body := c.lastChild()
+					if len(c.child) > 1 {
+						cond := c.child[0]
+						cond.tnext = body.start
+						if i == l-1 {
+							setFNext(cond, n)
+						} else {
+							setFNext(cond, clauses[i+1].start)
+						}
+						c.start = cond.start
 					} else {
-						setFNext(cond, clauses[i+1].start)
+						c.start = body.start
 					}
-					c.start = cond.start
-				} else {
-					c.start = body.start
-				}
-				// If last case body statement is a fallthrough, then jump to next case body
-				if i < l-1 && len(body.child) > 0 && body.lastChild().kind == fallthroughtStmt {
-					body.tnext = clauses[i+1].lastChild().start
-				} else {
-					body.tnext = n
+					// If last case body statement is a fallthrough, then jump to next case body
+					if i < l-1 && len(body.child) > 0 && body.lastChild().kind == fallthroughtStmt {
+						body.tnext = clauses[i+1].lastChild().start
+					} else {
+						body.tnext = n
+					}
 				}
 			}
 			sbn.start = clauses[0].start

